src/components/appBar.js

```diff
@@ -35,12 +35,13 @@
       setAnchorEl(null);
     };
 
-    const logoComponent = logo && <img src={logo} width="100" alt="" />;
-    const logoWithLink = endpoint.id ? (
+    const logo = Text({ value: logoLink });
+    const logoCmp = logo && <img src={logo} width="100" alt="" />;
```

Comment:
`logoCmp` should be written as `LogoCmp` because it is representing a React component.